### DISCUSSION TITLE
Repository > Issues - "Paste, Drop or click to add files" button is missing when editing an issue

### DISCUSSION THREAD

**Discussion by peter-angelov** [community/community#146917] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello,

My question is related to **Issues** section in any personal repository  (web interface). I first create a new issue (the top image in the screenshot) shows the new issue screen) and add some media (image or video) via **Paste, Drop or click to add files** button. 

Later on when I decide to edit the already created issue that contains any media this button is missing (the bottom image in the screenshot) so I can't replace the media (image or video) in any way. 

![github_issues_problem](https://github.com/user-attachments/assets/c738cff3-9076-46b0-ba81-cdc0468c053a)

So my question is - am I'm doing something wrong or just such functionality is not currently available?

**Comment by techy4shri** [comment, marked as answer]
The option appears in new issues and is not in the already created ones because now your issue is created! You don't need guidance (As per github) so you do it another way.
You can add new image or replace it by using the pin option beside the @ in the top right corner of the box. That pin button helps you attach files and you use it to add any media/whatever you want to upload.
Here:
<img width="605" alt="image" src="https://github.com/user-attachments/assets/b50d1dc8-74b7-4e4d-8ab6-4c5bfdbec453" />

Hope this helps! Please do mark as answer if it did. :)
